src/Symfony/Bundle/FrameworkBundle/Command/ConfigDebugCommand.php

```diff
@@ -98,4 +116,28 @@ private function compileContainer()
 
         return $container;
     }
+
+    /**
+     * Iterate over configuration until the last step of the given path.
+     *
+     * @param array $config A bundle configuration.
+     *
+     * @throws LogicException If the configuration does not exist
+     *
+     * @return mixed
+     */
+    private function getConfigForPath(array $config = array(), $path, $alias)
+    {
+        $steps = explode('.', $path);
```

Comment:
It would be nice but dot notation seems to be supported for objects only.

```
$array = array('foo' => array('bar' => 'value'));
$value = PropertyAccessor::getValue($arr, 'foo.bar');
```
GIves

> Cannot read property "foo" from an array. Maybe you intended to write the property path as "[foo]" instead.

It needs to be `PropertyAccessor::getValue($array, '[foo][bar]')`.
Is there a real gain to parse the propertyPath then wrap each level in `[level]` for finally use the property accessor?